projects/igniteui-angular/src/lib/date-picker/date-picker.component.spec.ts

```diff
@@ -1332,12 +1332,33 @@ describe('IgxDatePicker', () => {
             fixture.detectChanges();
             expect(inputDirective.valid).toEqual(IgxInputState.INITIAL);
 
-            inputDirectiveElement.triggerEventHandler('blur', { target: { value: ''}});
+            inputDirectiveElement.triggerEventHandler('blur', { target: { value: '' } });
             fixture.detectChanges();
 
             expect(inputDirective.valid).toEqual(IgxInputState.INVALID);
         }));
 
+        it('Should set date picker status to invalid when date is disabled', fakeAsync(() => {
```

Comment:
We could add an additional test that checks for essentially the same thing but with a user's template.

Reply:
done